### PR TITLE
Update SPO-on-boarding-guide.md, it's not required (anymore?) to run …

### PR DESCRIPTION
**Pull request by brouwerQ**
…Mithril on preprod before you can run it on mainnet

## Content

This PR includes...

## Pre-submit checklist

- Branch
  - [ ] Tests are provided (if possible)
  - [ ] Crates versions are updated (if relevant)
  - [ ] Commit sequence broadly makes sense
  - [ ] Key commits have useful messages
- PR
  - [ ] No clippy warnings in the CI
  - [ ] Self-reviewed the diff
  - [ ] Useful pull request description
  - [ ] Reviewer requested
- Documentation
  - [ ] Update README file (if relevant)
  - [ ] Update documentation website (if relevant)
  - [ ] Add dev blog post (if relevant)

## Issue(s)

Relates to #YYY or Closes #YYY

### CODE REVIEW

File: docs/website/versioned_docs/version-maintained/manual/getting-started/SPO-on-boarding-guide.md [hunk 1]
```diff
@@ -47,13 +47,13 @@ Each **Mithril network** has its own configuration, which can be found in the [C
 
 ## Step 2: Get Mithril ready for `mainnet`
 
-:::warning
+:::info
 
-This step is **mandatory**. You must complete it before moving forward to **Step 3**.
+This step is **optional**, but recommended.
```
Comment (jpraynaud) [inline docs/website/versioned_docs/version-maintained/manual/getting-started/SPO-on-boarding-guide.md:52]:
```suggestion
This step is **optional**, but **strongly recommended**.
```

File: docs/website/versioned_docs/version-maintained/manual/getting-started/SPO-on-boarding-guide.md [hunk 2]
```diff
@@ -47,13 +47,13 @@ Each **Mithril network** has its own configuration, which can be found in the [C
...
 
 :::
 
-To get ready for `mainnet`, you need to:
+To get ready for `mainnet`, you're encouraged to:
```
Comment (jpraynaud) [inline docs/website/versioned_docs/version-maintained/manual/getting-started/SPO-on-boarding-guide.md:56]:
```suggestion
To get ready for `mainnet`, you should:
```

File: docs/website/versioned_docs/version-maintained/manual/getting-started/SPO-on-boarding-guide.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ Each **Mithril network** has its own configuration, which can be found in the [C
 
 :::warning
 
-This step is **mandatory**. You must have completed **Step 2** before.
+This step is **mandatory**.
```
Comment (jpraynaud) [inline docs/website/versioned_docs/version-maintained/manual/getting-started/SPO-on-boarding-guide.md:71]:
```suggestion
This step is **mandatory**. It is **strongly recommended** to complete **Step 2** before.
```

**Review by jpraynaud — CHANGES_REQUESTED** [review]
Thanks @brouwerQ for this PR :slightly_smiling_face: 

I have made few suggestions as running on `preprod` is a strongly encouraged step before running on `mainnet`

File: docs/website/versioned_docs/version-maintained/manual/getting-started/SPO-on-boarding-guide.md (hunk 3 above)
Comment (olgahryniuk) [reply docs/website/versioned_docs/version-maintained/manual/getting-started/SPO-on-boarding-guide.md:71]: Suggest 'beforehand'

**Review by jpraynaud — APPROVED** [review]
LGTM :+1:

**Pull request merged**